lib/datasource/gitlab/index.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
   getPkgReleases,
 };
 
+const GitLabApiUrl = 'https://gitlab.com/api/v4/projects';
```

Comment:
Long-term it would be nice if this came from `config` so that Renovate could fetch presets from on-premise GitLab instances.

Reply:
@hutson thanks for the review, I think it totally makes sense.
@rarkins do you mind me to open an issue to address this in the future?

Reply:
Yes, make the issue “Allow self-hosted presets”. We need to consider hiwnthe namespacing would work. Right now GitHub> means GitHub.com and GitLab> means GitLab.com, similar to npm’s shortcuts

Reply:
@rarkins done. Here is the issue - https://github.com/renovatebot/renovate/issues/3019